ironfish/src/blockchain/blockchain.ts

```diff
@@ -1049,6 +1049,14 @@ export class Blockchain {
     return hash || null
   }
 
+  async getHashByTransactionHash(
```

Comment:
is `getBlockHashByTransactionHash` too wordy, or unnecessary given where the function lives and its return type? I ask because when I initially read this, until I saw the return type, I was confused, "wait, if we have the hash why do we need a hash, is this a no-op?"

Reply:
I think that `getBlockHashByTransactionHash` would be clearer

I left out the `Block` because there are a lot of other methods like `getHashAtSequence` that have names that leave the 'block' part implicit. Since there are two different hashes at play here I think it would be better to specify